wikipendium/user/forms.py

```diff
@@ -15,4 +15,4 @@ def clean(self):
 
 
 class EmailChangeForm(Form):
-    email = CharField(max_length=75, label='New email')
+    email = EmailField(max_length=75, label='New email')
```

Comment:
Nice catch!

I also believe the max length can be changed to 254 now, since we've migrated to Django 1.8
https://docs.djangoproject.com/en/1.8/releases/1.8/#default-emailfield-max-length-increased-to-254